rrfs-test/IODA/python/bufr2ioda_msonet_prepbufr.py

```diff
@@ -0,0 +1,562 @@
+
+    # Create IODA ObsSpace
+    iodafile = f"{cycle_type}.t{hh}z.{data_type}.tm00.api.nc"
+    OUTPUT_PATH = os.path.join(ioda_dir, iodafile)
+    logger.info(f"Create output file: {OUTPUT_PATH}")
+    obsspace = ioda_ospace.ObsSpace(OUTPUT_PATH, mode='w', dim_dict=dims)
+
+    # Create Global attributes
+    logger.debug(' ... ... Create global attributes')
+    obsspace.write_attr('sourceFiles', bufrfile)
+    obsspace.write_attr('description', data_description)
+
+    # Create IODA variables
+    logger.debug(' ... ... Create variables: name, type, units, and attributes')
+    # Observation Type: stationElevation
+    obsspace.create_var('ObsType/stationElevation', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_elv)
+
+    # Observation Type: airTemperature
+    obsspace.create_var('ObsType/airTemperature', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_tob)
+
+    # Observation Type: virtualTemperature
+    obsspace.create_var('ObsType/virtualTemperature', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_tvo)
+
+    # Observation Type: dewPointTemperature
+    obsspace.create_var('ObsType/dewPointTemperature', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_tdo)
+
+    # Observation Type: stationPressure
+    obsspace.create_var('ObsType/stationPressure', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_ps)
+
+    # Observation Type: specificHumidity
+    obsspace.create_var('ObsType/specificHumidity', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_qob)
+
+    # Observation Type: windEastward
+    obsspace.create_var('ObsType/windEastward', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_uob)
+
+    # Observation Type: windNorthward
+    obsspace.create_var('ObsType/windNorthward', dtype=obstyp.dtype, fillval=obstyp.fill_value) \
+        .write_attr('long_name', 'Observation Type') \
+        .write_data(typ_vob)
+
+    # MetaData: Latitude
+    obsspace.create_var('MetaData/latitude', dtype=lat.dtype, fillval=lat.fill_value) \
+        .write_attr('units', 'degrees_north') \
+        .write_attr('valid_range', np.array([-90, 90], dtype=np.float32)) \
+        .write_attr('long_name', 'Latitude') \
+        .write_data(lat)
+
+    # MetaData: Longitude
+    obsspace.create_var('MetaData/longitude', dtype=lon.dtype, fillval=lon.fill_value) \
+        .write_attr('units', 'degrees_east') \
+        .write_attr('valid_range', np.array([-180, 180], dtype=np.float32)) \
```

Comment:
Just putting a note here for when you comment the `lon[lon > 180]  -= 360` this also needs updated.